Marlin/src/HAL/HAL_STM32/timers.h

```diff
@@ -69,6 +73,7 @@
     #define TEMP_TIMER 14
   #endif
 
+  // TIM7 is consumed by Software Serial if used.
```

Comment:
Where is this specified? is it the default?

Reply:
That is the default. It is probably safer to set it explicitly, but you have to do it in the variant file or with -DTIMER_SERIAL=TIMx.